src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ";";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File readFile = new File(fromFileName);
+        File writeFile = new File(toFileName);
+        String fileContent = readFromFile(readFile);
+        MarketStatistic marketStatistic = getMarketStatistic(fileContent);
+        writeToFile(marketStatistic.getBuyCount(), marketStatistic.getSupplyCount(), writeFile);
+    }
+
+    private String readFromFile(File file) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(SEPARATOR);
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file" + file, e);
+        }
+        return builder.toString();
+    }
+
+    private MarketStatistic getMarketStatistic(String content) {
+        String[] listOfStatistic = content.split(";");
+        int supplyCount = 0;
+        int buyCount = 0;
+        for (String statistic : listOfStatistic) {
+            int statisticNumber = Integer.parseInt(statistic.split(",")[1]);
```

Comment:
```suggestion
            int statisticNumber = Integer.parseInt(statistic.split(LINE_SPLIT_DELIMITER)[NUMBER_INDEX]);
```
let's create corresponding constants